Resources/id.lproj/Localizable.strings

```diff
@@ -19,4 +19,4 @@
 "YPImagePickerVideoDurationTitle" = "Durasi video";
 "YPImagePickerVideoTooLong" = "Pilih durasi video kurang dari %@ detik";
 "YPImagePickerVideoTooShort" = " Durasi video setidaknya %@ detik";
-"YPImagePickerWarningItemsLimit" = "Hanya diperbolehkan %d foto atau video";
```

Comment:
Let's try not to include unrelated changes, I know Xcode loves to change things without us knowing ;)

Reply:
This is Localize.swift made this changes. I think it would do it all the time, so better to commit once and not pay attention on this in future.

Reply:
👍